File: week6/py/secret_lab.py
s = input()
cur = s[0]
count = 1
max_count = 0

for i in range(1, len(s)):
    if s[i] == cur:
        count += 1
    else:
        cur = s[i]
        max_count = max(max_count, count)
        count = 1

max_count = max(max_count, count)
print(max_count)

File: week6/cpp/secret_lab.cpp
#include <bits/stdc++.h>
#include <cstdint>
using namespace std;
/* #define int long long */

int main() {
  string s;
  cin >> s;
  char cur = s[0];
  int count = 1, max = 0;
  for (int i = 1; i < s.size(); i++) {
    if (s[i] == cur)
      count++;
    else {
      cur = s[i];
      max = (max > count) ? max : count;
      count = 1;
    }
  }
  cout << ((max > count) ? max : count);
  return 0;
}
